### ISSUE TITLE
Get rid of courses/courses_taught, departments, majors/minors, and remove userinfo/userinfo_update

### ISSUE THREAD

**Issue by sungbae** [SFII/scq#149]
I'll create a separate PR for this real quick.

**Comment by sungbae** [comment]
Someone already got rid of courses/courses_taught, departments, majors/minors. My recent PR will remove userinfo/userinfo_update #163

**Comment by Brady76** [comment]
@sungbae remove this when you're ready

**Comment by sungbae** [comment]
I'll remove this once the PR #163 is merged in.
